marquez_client/constants.py

```diff
@@ -14,4 +14,4 @@
 DEFAULT_NAMESPACE_NAME = 'default'
 DEFAULT_MARQUEZ_BACKEND = 'file'
 DEFAULT_MARQUEZ_URL = 'http://localhost:5000'
-DEFAULT_MARQUEZ_FILE = '/tmp/marquez.client.requests.log'
+DEFAULT_MARQUEZ_FILE = '/tmp/marquez/client.requests.log'
```

Comment:
Do we need to make sure the folder exists before creating the file? I think the previous default would work out of the box while this one will fail unless /tmp/marquez has been created.
Somewhere here? 
https://github.com/MarquezProject/marquez-python/blob/main/marquez_client/file_backend.py#L23

Reply:
Yeah, I looked into it briefly, but we can use [`os.makedirs`](https://docs.python.org/3/library/os.html#os.makedirs) before we open the file to append.

Reply:
LGTM.
security / permissions, for later.